rnaseq/workflow.cwl

```diff
@@ -36,14 +36,20 @@ inputs:
         type: int
     trimming_min_readlength:
         type: int
-    kallisto_index: 
+    kallisto_index:
        type: File
     gene_transcript_lookup_table:
        type: File
     firststrand:
        type: boolean?
     secondstrand:
        type: boolean?
+    refFlat:
+        type: File
+    ribosomal_intervals:
+        type: File
+    strand:
```

Comment:
Could we calculate this value from the existing `firststrand` and `secondstrand` inputs?

Reply:
If possible, calculating this property would avoid much of the confusion associated with strand. For example, if it's the firststrand setting in StringTie, then you'd want SECOND_READ_TRANSCRIPTION_STRAND for this tool. 

Here is a summary table from the RNA-seq PLoS paper with the Griffith Lab:
https://github.com/griffithlab/rnaseq_tutorial/blob/e6677df4f76c7c023c725de6202cb9e421aa459a/manuscript/supplementary_tables/supplementary_table_5.md